tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/display.py

```diff
@@ -9,7 +9,7 @@
 
 class Display(Base):
 
-    _page_locator = (By.ID, 'sound')
```

Comment:
Wait, I don't think this line was there  for 2 years- I think I added it during my RTL work recently (https://github.com/mozilla-b2g/gaia/commit/7a0ebe3d2ffac3162417b92c57c272286b2b5673#diff-d6a18b8bf566e3f2ed25c705ae628f7f), I found this at the same time martijn did, and he corrected it while testing it on the entire RTL test